docs/DesignMDIngestion.md

```diff
@@ -0,0 +1,63 @@
+# *Generic Metadata Ingestion*
```

Comment:
Why italicize the title?

Reply:
:fearful: I was moving around components in a template - I'll remove this though!